<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FUsersAdminPage</class>
 <widget class="QRecordWidgetMapper" name="FUsersAdminPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecordWidgetMapper</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Usuarios:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TListViewModelRecord" name="lvwUsuarios">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnNuevo">
         <property name="toolTip">
          <string>Nuevo usuario</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/list-add.png</normaloff>:/pics/list-add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEliminar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Eliminar usuario</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/list-remove.png</normaloff>:/pics/list-remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAtras">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Usuarios anteriores</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/atras.png</normaloff>:/pics/atras.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdelante">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Siguientes usuarios</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../app/app.qrc">
           <normaloff>:/pics/adelante.png</normaloff>:/pics/adelante.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="grbxUsuarioDatos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Usuario datos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="5">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Usuario:</string>
          </property>
          <property name="buddy">
           <cstring>ledUsuario</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ledUsuario"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbltEmpresa">
          <property name="text">
           <string>Empresa:</string>
          </property>
          <property name="buddy">
           <cstring>cbxEmpresa</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="TComboBoxModelRecord" name="cbxEmpresa"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbltNombres">
          <property name="text">
           <string>Nombres:</string>
          </property>
          <property name="buddy">
           <cstring>ledNombres</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="ledNombres"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblApellidos">
          <property name="text">
           <string>Apellidos:</string>
          </property>
          <property name="buddy">
           <cstring>ledApellidos</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ledApellidos"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblRol">
          <property name="text">
           <string>Rol:</string>
          </property>
          <property name="buddy">
           <cstring>cbxRol</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="TComboBoxModelRecord" name="cbxRol"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblCargo">
          <property name="text">
           <string>Cargo:</string>
          </property>
          <property name="buddy">
           <cstring>cbxCargo</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="TComboBoxModelRecord" name="cbxCargo"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnCancelar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset resource="../../app/app.qrc">
          <normaloff>:/pics/edit-delete.png</normaloff>:/pics/edit-delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnModificarClave">
        <property name="text">
         <string>Modificar contrase√±a</string>
        </property>
        <property name="icon">
         <iconset resource="../../app/app.qrc">
          <normaloff>:/pics/document-edit.png</normaloff>:/pics/document-edit.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnGuardar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
        <property name="icon">
         <iconset resource="../../app/app.qrc">
          <normaloff>:/pics/save.png</normaloff>:/pics/save.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TComboBoxModelRecord</class>
   <extends>QComboBox</extends>
   <header>tcomboboxmodelrecord.h</header>
  </customwidget>
  <customwidget>
   <class>TListViewModelRecord</class>
   <extends>QListView</extends>
   <header>tlistviewmodelrecord.h</header>
  </customwidget>
  <customwidget>
   <class>QRecordWidgetMapper</class>
   <extends>QWidget</extends>
   <header>qrecordwidgetmapper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../app/app.qrc"/>
  <include location="../../app/app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FUsersAdminPage</sender>
   <signal>hasRecordsWithOutSave(bool)</signal>
   <receiver>btnGuardar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FUsersAdminPage</sender>
   <signal>hasRecordsWithOutSave(bool)</signal>
   <receiver>btnCancelar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
